backend/src/main/java/com/summerHack/diningTogether/service/UserService.java

```diff
@@ -75,16 +79,23 @@ public UserDetails getUserDetailsByUsername(String username) throws UsernameNotF
      * @throws UserAlreadyExistException User with the username or email exist
      */
     @Transactional
-    public User registerUser(RegisterInput input) throws UserAlreadyExistException {
+    public User registerUser(RegisterInput input, String siteURL) throws UserAlreadyExistException, MessagingException {
```

Comment:
Usually we use a separate method to send verification emails in case that the user want to resend it.